### ISSUE TITLE
VST3 Plugin is not showing up in DAW (macOS)

### ISSUE THREAD

**Issue by rjbunge99** [Torsion-Audio/Scyclone#3]
got to the point where scyclone.vst3 was in my VST3 folder. Opened in my DAW, plugin would not scan. The folder seems empty, it's only taking up 17KB of storage. Not sure what's missing.

I know I didn't give enough information for anyone to help me. Just stating that I found the installation guide hard to follow. i'm not sure what certain things mean, in what order I'm supposed to do stuff, and right when I thought i figured it out, the vst3 folder is empty. I'm sure I did it all wrong. Just rambling. sorry.

Plugin looks cool on Youtube!

**Comment by vackva** [comment]
Hi @rjbunge99, thank you for contacting us about this issue. 

In order to better understand the issue, could you please provide some additional information?

1. Did you use the binaries provided under release or did you build the plugin yourself with cmake?
2. Are you using an Intel or M1/M2 based system?
3. Did you install onnxruntime via homebrew as explained in the README?
4. Is the standalone version running?

**Keep in mind (written in the README):**
> (On macOS) The provided binaries are not notarized yet. As a result, your DAW may be unable to load the VST plugin due to your system's security settings. In such cases, you may need to either build the plugin yourself or adjust your security settings accordingly.

We will be releasing v0.0.3 soon with a static link to onnx (current development branch - macOS support only at the moment). This will make the installation process much easier. With v.0.0.3 will also notarize the plugins, so our provided binaries of the vst plugin should work then

**Comment by vackva** [comment]
problem is solved with the newest release

https://github.com/Torsion-Audio/Scyclone/releases/tag/v.0.0.3
